rclcpp/include/rclcpp/rate.hpp

```diff
@@ -112,8 +120,56 @@ class GenericRate : public RateBase
   std::chrono::time_point<Clock, ClockDurationNano> last_interval_;
 };
 
-using Rate = GenericRate<std::chrono::system_clock>;
-using WallRate = GenericRate<std::chrono::steady_clock>;
+class Rate : public RateBase
```

Comment:
Because this is no longer templated, on Windows I think we need to add the `PUBLIC` keyword:

```suggestion
class RCLCPP_PUBLIC Rate : public RateBase
```

Reply:
(if that doesn't work on Windows, then the other option is to add `RCLCPP_PUBLIC` to each of the methods in the class)